pbspro.spec.in

```diff
@@ -403,6 +406,7 @@ fi
 
 %postun %{pbs_server}
 if [ "$1" != "1" ]; then
+	ldconfig %{_libdir}
```

Comment:
Please place these lines below the comment regarding uninstall/upgrade. That's why we test $1